apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  styled,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioLogoFlatIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { useState, type ComponentProps } from "react";
+
+const StyledLink = styled("a", {
+  textDecoration: "none",
+  color: "white",
+  display: "inline-flex",
+  cursor: "default",
+});
+
+const PopoverItemButton = ({
+  href,
+  ...buttonProps
+}: ComponentProps<typeof Button> & { href?: string }) => (
+  <StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
+    <Button
+      css={{
+        width: "100%",
+      }}
+      {...buttonProps}
+    />
+  </StyledLink>
+);
+
+export const HelpPopover = ({
+  children,
+  side = "right",
+}: {
+  children: React.ReactNode;
+  side?: "bottom" | "left" | "right" | "top";
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Popover open={isOpen} onOpenChange={setIsOpen}>
+      {children}
+      <PopoverContent
+        side={side}
+        css={{
+          zIndex: theme.zIndices["max"],
+          boxShadow: theme.shadows.menuDropShadow,
+          background: "white",
+          border: theme.colors.borderMain,
```

Comment:
all of this should not be necessary, design system has to take care of this

Reply:
if popover design is not aligned with design system in figma, we have to fix popover, not change it adhoc here

Reply:
I would not fix popover as part of this pr though, definitely a separate one, this pr can be merged with the slight difference in popover styling compared to figma

Reply:
Ok then I'll remove the designs.

Reply:
I dont know I can't find any other popover variants.

Reply:
There are no variants, it's just the singular popover component. @kof can you help Wasim identify the component in the code?

Reply:
@taylornowotny Other than these two everything else looks good?

I didn't overlook the designs in terms of spacing. So let me know if there's any changes needed there

Reply:
<img width="715" alt="Screenshot 2023-08-23 at 10 03 49 AM" src="https://github.com/webstudio-is/webstudio-builder/assets/86495896/f220324a-0efb-4db4-87a0-aa3fedbae5a1">

Reply:
I'm using the popover component from the design system.

What about the above image?

Reply:
@taylornowotny Can you check now for the offset?

Reply:
If you can also tell me what's the issue with the dashboard popover, I can fix it right away

Reply:
If it looks like the screen you posted above then it's good.
I am not able to check becuase the preview link that I was using isn't working any more and I can't find a new one @kof 

The offset doesn't look like 8px but it's close enough.

The dashboard popover needed the same changes - use the popover component and position it 8px from the button. When that is done then it is good.